react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,26 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+const returnOneArray = (array) => array.reduce((acc, val) => acc.concat(val), []);
+
+function min(...params) {
+  if (params.length === 0) return undefined;
+
+  return isArray(...params)
+    ? Math.min(...returnOneArray(params))
+    : Math.min(...params);
+}
+
+function copy(copiedArray) {
```

Comment:
As a side note, I wouldn't name this `copiedArray` since the argument is not a copied array. You can name this argument `element` for example, because you don't know what's the type of the argument.